src/demo/models/guide.js

```diff
@@ -0,0 +1,299 @@
+import {getState, setState} from '../state';
+import {AppMode, Modes} from '../constants';
+
+const guides = [
+  {
+    id: 'fishvtrash-training-init',
+    text: 'Welcome to the tutorial!',
+    when: {appMode: AppMode.FishVTrash, currentMode: Modes.Training},
+    style: 'TopLeft',
+    arrow: 'none'
+  },
+  {
+    id: 'fishvtrash-training-init2',
+    text: 'Use these two buttons to train A.I.',
+    when: {appMode: AppMode.FishVTrash, currentMode: Modes.Training},
+    style: 'BottomMiddleButtons'
+  },
+  {
+    id: 'fishvtrash-training-done',
+    text: "Great work!  You can continue when you're ready.",
+    when: {
+      appMode: AppMode.FishVTrash,
+      currentMode: Modes.Training,
+      fn: state => {
+        return state.yesCount + state.noCount > 15; // TODO: 40
```

Comment:
Seeing that this is mostly something we could serialize to JSON makes me want us to load it dynamically and have a tool for tweaking this script without rebuilding the bundle.

Reply:
Good point.  It also struck me as a feature that long-term could be hoisted out into data, rather than code, especially if we wanted to build an authoring UI for it.  A nice thing about the current implementation is that our reload times for the app are really fast, and so iteration on this has been very easy.  Though you do lose your place in the UI when the app reloads.

Reply:
The way this is built should also hopefully make it easier for localization at some point.